### ISSUE TITLE
Provide CheckoutActions with addressBook data if available 

### ISSUE THREAD

**Issue by nnnnat** [reactioncommerce/example-storefront#391]
Type: **major**

**Describe the bug**
We need to pass down `account.addressBook` data to the `CheckoutActions` component if a customer is logged in. Currently, we're defaulting to the first option in the address book. 

**To Reproduce**
1. In the Storefront login as a customer with saved address then checkout.
2. During the checkout process, the shipping address should default to the first address in the address book with only the option of editing the current one.

**Expected behavior**
As a logged in customer with a saved address I want the ability to select a shipping address from those options.

**Additional context**
Blocked by comp-lib [#332](https://github.com/reactioncommerce/reaction-component-library/issues/332)

**Comment by HarisSpahija** [comment]
Duplicate of #554 This can be closed and moved to one issue
